pkgs/top-level/aliases.nix

```diff
@@ -1307,6 +1307,7 @@ mapAliases {
   unifi8 = unifi; # Added 2024-11-15
   unifiLTS = throw "'unifiLTS' has been removed since UniFi no longer has LTS and stable releases. Use `pkgs.unifi` instead."; # Added 2024-04-11
   unifiStable = throw "'unifiStable' has been removed since UniFi no longer has LTS and stable releases. Use `pkgs.unifi` instead."; # Converted to throw 2024-04-11
+  unl0kr = lib.warnOnInstantiate "'unl0kr' is now deprecated, as it's now included in buffybox." buffybox; # Deprecated since 2024-12-07
```

Comment:
```suggestion
```
And wrap the entire derivation in [`unl0kr/package.nix`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name/un/unl0kr/package.nix) under `lib.warn`, similar to [this](https://github.com/NixOS/nixpkgs/blob/2f7f4f9432a94d75709db956dd19f6a91f7166a9/pkgs/by-name/pa/pay-respects/package.nix#L13).